content/exchange/finance_us.md

```diff
@@ -0,0 +1,9 @@
+---
+image: >-
+  https://res.cloudinary.com/duquzk7m6/image/upload/v1665428971/finance-us_dtnyn8.png
```

Comment:
Also, it looks like this image isn't resolving properly

Reply:
I actually don't know what the logo is (I've tried googling it, but was provided with gray circle in designs) - I've updated in latest to closest thing I could find